### ISSUE TITLE
Messes up with the `~` character

### ISSUE THREAD

**Issue by atishay** [JPeer264/node-rename-css-selectors#48]
When we have a css like this:
```css
.test,
.test~.icon {
display: none;
}
```
In the mapping, it generates:

```
".test": "a",
".test~":, "b"
```

**Comment by JPeer264** [comment]
Oh ya this is a bug indeed. I'm gonna fix this asap :)

**Comment by JPeer264** [comment]
Should be fixed in v`3.2.6` (ref: https://github.com/JPeer264/node-rcs-core/commit/09eb21b5a4d81c6a514655411bc551c61f6425ef)

**Comment by JPeer264** [comment]
I had to reopen this issue, as I had to unpublish this version as it is flaky (ref: https://github.com/JPeer264/node-rename-css-selectors/pull/44#issuecomment-558426080). I am very sorry about this, but I hope this is fixed asap :)

**Comment by JPeer264** [comment]
Fixed in v3.2.8
